docs/cody/clients/model-configuration.mdx

```diff
@@ -0,0 +1,518 @@
+# LLM Model Configuration
+
+<p className="subtitle">This guide will walk you through the steps to customize the LLM models available from your Sourcegraph Enterprise instance.</p>
+
+<Callout type="note">For Sourcegraph Cloud customers, configuring the available LLM models requires contacting your Sourcegraph account team representative</Callout>
+
+Starting with Sourcegraph 5.5.4+ Sourcegraph Enterprise supports the ability to choose between different LLM models, allowing developers to use the best model for Cody Chat as needed.
```

Comment:
I believe the release on Aug 7th was 5.6.0, right?

Reply:
Yes, it was. Good catch.